### DISCUSSION TITLE
UnoCSS Support

### DISCUSSION THREAD

**Discussion by louiss0** [zed-industries/zed#6608] [Feature Requests]
Now that taliwind CSS is supported can we get support for UnoCSS. Read [here](https://unocss.dev/). It's a CSS engine that is used
to generate classes based on your utilities. It's a tool that uses presets to define rules for what and how many CSS classes should be generated. It's heavily used by the Vue community.

**Comment by reslear** [comment]
https://github.com/bajrangCoder/zed-unocss

**Reply by louiss0** [reply]
Thanks!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
